### ISSUE TITLE
[WPF, NET8, Lvc2.0.0-rc2] New separators have wrong opacity when they appear

### ISSUE THREAD

**Issue by foglio1024** [beto-rodriguez/LiveCharts2#1395]
**Describe the bug**
New separators and ticks have full opacity when they appear, even if a Paint with lower opacity is specified when creating the Axis. The correct opacity is applied only after the animation ends. The issue doesn't happen when the chart is not animated.

**To Reproduce**
1. Create a CartesianChart and ViewModel like this
```xaml
            <wpf:CartesianChart XAxes="{Binding XAxes}"
                                YAxes="{Binding YAxes}"
                                Series="{Binding Series}"
                                AnimationsSpeed="0:0:0.5"/>
```

```cs

// ViewModel.cs

public Axis[] XAxes { get; }
public Axis[] YAxes { get; }
public ISeries[] Series { get; }

public ViewModel()
{
   XAxes = [
      new Axis
        {
            LabelsPaint = new SolidColorPaint(SKColors.DimGray.WithAlpha(100)),
            SeparatorsPaint = new SolidColorPaint(SKColors.DimGray.WithAlpha(100))
        }
    ];
    // same for YAxes
   
   Series = [
      new LineSeries<ObservablePoint>
        {
             // [...]
        }
   ];
}


```

2. Add random points to the LineSeries every 1 second.
3. Observe new separators coming in as the new points push the chart boundaries
4. As separators and labels come in, they are full opacity DimGray and suddenly change to the desired value (Alpha = 100) after the animation ends

**Expected behavior**
The separators and labels should already have the desired alpha from the start of the animation.

**Screenshots**
Short gif showing the issue:
![ezgif-3-3e21dba137](https://github.com/beto-rodriguez/LiveCharts2/assets/18739596/8da867ca-b9bb-4d1c-8251-a745d12c66e1)

**Desktop (please complete the following information):**
 - OS: Windows 11 22H2 (build 22621.2715)
 - Platform:  WPF and .NET 8 (also happened on .NET6 and 7)
 - LVC2 version: 2.0.0-rc2 (the issue was present in older beta versions too)

**Comment by beto-rodriguez** [comment]
Hello and thanks for the report.

I will close this as duplicate of #1344.

The good news is that this is already fixed and will be included in the next version of the library!
